src/components/KTreeList/KTreeItem.vue

```diff
@@ -83,6 +104,31 @@ const handleClick = (event: any) => {
 
   emit('selected', props.item)
 }
+
+const isExpanded = ref<boolean>(true)
+
+const expandedIcon = shallowRef(ChevronUpIcon)
+
+const setExpandedIcon = (): void => {
+  expandedIcon.value = isExpanded.value ? ChevronUpIcon : ChevronDownIcon
```

Comment:
When the item is collapsed, we want the chevron to be pointed to the right rather than up